.github/workflows/ci.yml

```diff
@@ -35,15 +35,24 @@ jobs:
   wasm-node-check:
     name: Check javascript node
     runs-on: ubuntu-latest
-    container:
-      image: rust
     steps:
+    - name: Start Polkadot node in the background
+      # Note: feel free to update the Polkadot version here when one is released
+      run: docker run --network host -d parity/polkadot:v0.9.8 --chain polkadot-dev --unsafe-rpc-external --unsafe-ws-external
+    - name: Wait for node to be up
+      run: |
+        bash -c 'while true; do curl -X POST -H "Content-Type: application/json" localhost:9933; if [ $? -eq 0 ]; then break; fi; sleep 1; done'
     - uses: actions/checkout@v2.3.4
+    - uses: actions-rs/toolchain@v1
+      with:
+        # Ideally we don't want to install any toolchain, but the GH action doesn't support this.
```

Comment:
`ubuntu-latest` has rust stable built-in

Reply:
you don't need this job

Reply:
I need `rustup` though, not just `rust`

Reply:
You also [have it](https://github.com/paritytech/cumulus/blob/master/.github/workflows/docs.yml#L18) there